eng/validation/test-publishing.ps1

```diff
@@ -0,0 +1,66 @@
+Param(
+  [Parameter(Mandatory=$true)][string] $buildId,
+  [Parameter(Mandatory=$true)][string] $azdoToken,
+  [Parameter(Mandatory=$true)][string] $barToken
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference = 'Stop'
+
+. $PSScriptRoot\..\common\darc-init.ps1
+
+$global:buildId = $buildId
+$global:targetChannel = "General Testing"
+$global:azdoToken = $azdoToken
+$global:barToken = $barToken
+
+function Find-BuildInTargetChannel(
+    [string] $buildId,
+    [string] $targetChannelName
+)
+{
+    $build = darc get-build --id $buildId --azdev-pat $global:azdoToken --password $global:barToken
```

Comment:
You can do this even better! (now)  --output-format json spits out the build info in json format which is then super easy to query.

Reply:
Ricardo beat you to it 😄

Reply:
He did, but did you hear that you can use --output-format json and all problems are solved?

Reply:
I really think we should be using --output-format json for this

Reply:
```
{
  "comment" : "json is the best format"
  "author" : "mmitche"
}
```

Reply:
Saved 12 lines of code by switching to JSON

![gecko](https://qph.fs.quoracdn.net/main-qimg-3da23224e6f8055f20800f63e9006ce7)